client/src/entrypoints/admin/page-editor.js

```diff
@@ -1,5 +1,5 @@
 import $ from 'jquery';
-import { cleanForSlug } from '../../utils/cleanForSlug';
```

Comment:
Can we settle on (and document) a convention before changing this please? The current de-facto convention of "use camelcase for things that get imported into other JS, and kebab-case for endpoints" feels like it makes sense to me.

Reply:
ok, I was not aware of that convention, kind of makes sense when I look at the files again (ish).

I am happy to rework this PR adopting that convention to avoid having to get a decision on that yet - it still feels like a bit of a confusing situation to me.

We also use `lower_snake_case` in one scenario for entrypoints `client/src/entrypoints/contrib/typed_table_block/typed_table_block.js`

And `client/src/includes/tabs.js` even though it exports `initTabs` but `client/src/includes/initTooltips.js` exports `initTooltips` along with `initModernDropdown`.

I have flagged this question with Thibaud on the draft UI development docs.

Reply:
Done - kept as `cleanForSlug` and not `clean-for-slug`.